cmd/mount/mount.go

```diff
@@ -73,6 +74,29 @@ func mountOptions(device string) (options []fuse.MountOption) {
 // report an error when fusermount is called.
 func mount(f fs.Fs, mountpoint string) (*vfs.VFS, <-chan error, func() error, error) {
 	fs.Debugf(f, "Mounting on %q", mountpoint)
+
```

Comment:
Can you factor this bit of code into a function and place it in cmd/mountlib/mount.go - say checkMountEmpty() or something like that.

Then you can call it [about here](https://github.com/ncw/rclone/blob/ed91d6b5a5ffd71c58572d0723afb24554aa2935/cmd/mountlib/mount.go#L140) and the check will work for both mount and cmount (cmount uses libfuse but is called mount on windows)

Reply:
No problem. I will do it.